documentation/modules/rn-2.3.adoc

```diff
@@ -71,3 +71,9 @@ The error status message for a VM with no operating system on the *Migration pla
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.{rhv-short} destination VMs fail to boot if source keys are encrypted with TPM
```

Comment:
the TPM part would be relevant as from the next version (2.4) so let's leave it out for now, what you wrote in jira better fits for 2.3.4: Migration of virtual machines with encrypted partitions fails during conversion